### ISSUE TITLE
Send to Device (Bluetooth) - Nemo Action

### ISSUE THREAD

**Issue by anandrkris** [linuxmint/nemo#819]
It is bit cumbersome to use Bluetooth Applet while one has to browse the file by traversing through directory. Can we a Nemo action (when bluetooth is enabled) to send files to device by using the Context Menu? We could also have a 'Send to' that further opens up options like Mail, Bluetooth,etc

Maybe, this is possible using 'Copy to' when device is mounted as file system, haven't checked it. Anyways, I don't have bluetooth device auto-mounted.

**Comment by ACinnamonUserInUK** [comment]
+1 (but this bug report at least comes close to duplicating this one: #906).

**Comment by leigh123linux** [comment]
-1 for this being added to nemo itself, it should be implemented as a nemo-extension instead.

**Comment by johanvdp** [comment]
I just searched the web to solve the same itch. Solved it in 2 minutes adding a simple nemo action. It depends on blueman-sendto to do the job. Just throw the following as file (I named it blueman-sendto.nemo_action) into (in my case (using fedora, sorry) /usr/share/nemo/actions:
[Nemo Action]
Name=Send to bluetooth
Comment=Send to bluetooth: %F
Exec=blueman-sendto %F
Icon-Name=bluetooth
Selection=any
Extensions=nodirs;
Quote=double
Dependencies=blueman-sendto;
